docs/docsite/rst/dev_guide/testing_documentation.rst

```diff
@@ -29,7 +29,7 @@ To ensure that your module documentation matches your ``argument_spec``:
 
    .. code-block:: bash
 
-      pip install --user -r test/runner/requirements/sanity.txt
+      pip install --user -r test/sanity/requirements.txt
```

Comment:
I recommend directing users to use either the `--docker` or `--venv` options. If running in their own venv, using `--requirements` also works. If requirements are to be installed manually then things get more complicated, as there are multiple requirements files involved, depending on the tests being executed and whether testing is being done on ansible/ansible or an Ansible Collection.